src/main/java/seedu/toluist/model/CommandHistoryList.java

```diff
@@ -0,0 +1,50 @@
+package seedu.toluist.model;
+
+import java.util.ArrayList;
+
+public class CommandHistoryList {
+    private ArrayList<String> commandHistory;
+    private int historyPointer = 0;
+    private String currentCommand;
+
+    public CommandHistoryList() {
+        commandHistory = new ArrayList<String>();
+        currentCommand = "";
+    }
+
+    public ArrayList<String> getAsArrayList() {
```

Comment:
change to getCommandHistoryList or getListOfCommandHistory or getCommandHistory?